pmoves/services/agentgym-rl-coordinator/coordinator/publisher.py

```
@@ -0,0 +1,373 @@
+                f"{self.supabase_url}/rest/v1/agentgym_trajectories",
+                headers=self._headers,
+                params={
+                    "session_id": f"eq.{session_id}",
+                    "select": "*",
+                },
+            )
+            if resp.status_code == 200:
+                trajectories = resp.json()
+
+        if not trajectories:
+            raise ValueError("No trajectories found")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)